src/core/api/models/technologies.ts

```diff
@@ -2,4 +2,5 @@ export interface Technology {
   id: number;
   name: string;
   description?: string;
+  discriminator: 'technologies';
```

Comment:
If this is property not from backend - just added by you for UI purposes - please delete that in every model